src/components/Footer/Footer.js

```diff
@@ -39,20 +41,20 @@ const Footer = () => {
           className={classes.link}
         >
           <i className={`fab fa-github ${classes.linkIcon}`}></i>
-          <span>source</span>
+          <span>{t('source')}</span>
         </a>
       </div>
 
       <div className={classes.column}>
-        <div className={classes.title}>products</div>
+        <div className={classes.title}>{t('products')}</div>
         <a
           href="https://gov.beefy.finance"
           target="_blank"
           rel="noopener noreferrer"
           className={classes.link}
         >
           <i className={`fas fa-landmark ${classes.linkIcon}`}></i>
-          <span>gov</span>
+          <span>{t('gov')}</span>
```

Comment:
here gov represents a subdomain, so translating it is not a good idea

Reply:
I'm going to fix them asap and get back to you.